### PR TITLE
olm cluster role fix

### PR DESCRIPTION
**Pull request by msenmurugan**
* Incorporated #266 
* Incorporated operator cluster role and role binding hard code values

### CODE REVIEW

**Comment by operator-r2d2** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by:
To fully approve this pull request, please assign additional approvers.
We suggest the following additional approver: **msenmurugan**

If they are not already assigned, you can assign the PR to them by writing `/assign @msenmurugan` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=blackducksoftware%2Fsynopsys-operator).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/blackducksoftware/synopsys-operator/blob/master/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

File: pkg/util/common.go [hunk 1]
```diff
@@ -1246,5 +1246,51 @@ func IsPodReady(clientset *kubernetes.Clientset, namespace string, labelSelector
...
+
+// GetOperatorNamespace returns the namespace of the synopsys operator based on the labels
+func GetOperatorNamespace(clientset *kubernetes.Clientset) (string, error) {
+	// check if operator is already installed
+	rcs, rcErr := ListReplicationControllers(clientset, "", "app=synopsys-operator,component=operator")
+	deployments, dErr := ListDeployments(clientset, "", "app=synopsys-operator,component=operator")
+	if rcErr != nil || dErr != nil || (len(deployments.Items) == 0 && len(rcs.Items) == 0) {
+		return "", fmt.Errorf("synopsys operator namespace not found")
+	}
+	return deployments.Items[0].Namespace, nil
```
Comment (rrati) [inline pkg/util/common.go]: Need to check which list to return right?  Either RC or Deployments, but currently you are only returning deployments.  Should check with list is not empty and return that one.

**Comment by operator-r2d2** [comment]
[APPROVALNOTIFIER] This PR is **APPROVED**

This pull-request has been approved by: *<a href="https://github.com/blackducksoftware/synopsys-operator/pull/270#pullrequestreview-233473055" title="Approved">rrati</a>*

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=blackducksoftware%2Fsynopsys-operator).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details >
Needs approval from an approver in each of these files:

- ~~[OWNERS](https://github.com/blackducksoftware/synopsys-operator/blob/master/OWNERS)~~ [rrati]

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

File: pkg/util/common.go (hunk 1 above)
Comment (msenmurugan) [reply pkg/util/common.go]: nice catch. Fixed it

**Pull request merged**